app/core/models/ocelot/stanzas/CalculationStanza.scala

```diff
@@ -54,63 +58,73 @@ sealed trait Operation {
   val right: String
   val label: String
 
-  def eval(labels: Labels) : Labels
+  def eval(labels: Labels): Labels
 
-  def value(arg: String, labels: Labels): String = labelReference(arg).fold(arg){ref => labels.value(ref).getOrElse("")}
+  def value(arg: String, labels: Labels): String = labelReference(arg).fold(arg) { ref =>
+    labels.value(ref).getOrElse("")
+  }
 
   def toPlainString(value: BigDecimal): String = value.bigDecimal.toPlainString
 
-  def addString(s1: String, s2: String) : Option[String] = Option(s1 + s2)
+  def addString(s1: String, s2: String): Option[String] = Option(s1 + s2)
 
-  def unsupportedOperation(s1: String, s2: String) : Option[String] = {
+  def unsupportedOperation(arg1: Any, arg2: Any): Option[String] = {
```

Comment:
opportunity for a curried fn 
`def unsupportedOperation(opName: String)(arg1: Any, arg2: Any): Option[String]`

Reply:
Done